src/gui/progressbarpainter.h

```diff
@@ -29,23 +29,17 @@
 #pragma once
 
 #include <QProgressBar>
-#include <QStyledItemDelegate>
 
-class QStyleOptionProgressBar;
+class QStyleOptionViewItem;
 
-class ProgressBarDelegate : public QStyledItemDelegate
+class ProgressBarPainter
```

Comment:
You probably should mark it as `final`.

Reply:
It isn't involved in inheritance so that would be meaningless IMO.

Reply:
Isn't it purposed to mark a class that is not intended to be inherited, even if it doesn't inherit other one itself?

Reply:
However, I do not insist in this case.